### ISSUE TITLE
DataLayer definition for multi-channel input

### ISSUE THREAD

**Issue by liguanbin** [BVLC/caffe#586]
Hi, all, I am trying to use caffe to do classification base on some extracted feature.

I extract feature of 87 dimensions for each image (say, including the color histogram, edge saliency...). And each image has a label 0 or 1.  I have already divide the data into the training set and the testing set.  I have no idea how to write the  protobuf definitions. 

here is the protobuf definition for mnist demo.
name: "LeNet"
layers {
  name: "mnist"
  type: DATA
  top: "data"
  top: "label"
  data_param {
    source: "D:\gbli\caffe\caffe-windows\examples\mnist\mnist-train-leveldb"
    scale: 0.00390625
    batch_size: 64
  }
}

so my problem is how should i set for data_param in my example?
